### PR TITLE
add in clause at FetchableSubQueryBase

### PR DESCRIPTION
**Pull request by songsungkyun**
New PR of https://github.com/querydsl/querydsl/pull/1749.
Because I deleted old #1749 PR repo by mistake.

### CODE REVIEW

File: querydsl-core/src/main/java/com/querydsl/core/support/FetchableSubQueryBase.java [hunk 1]
```diff
@@ -153,4 +156,18 @@ public final QueryMetadata getMetadata() {
...
+        } else {
+            return Expressions.booleanOperation(Ops.IN, mixin, ConstantImpl.create(right));
+        }
+    }
+
+    @Override
+    public BooleanExpression in(T... right) {
+        return this.in(Arrays.asList(right));
+    }
+}
```
Comment (Shredder121) [inline querydsl-core/src/main/java/com/querydsl/core/support/FetchableSubQueryBase.java]: I noticed that this line doesn't end with a newline.

**Comment by Shredder121** [comment]
One more comment, but to me it looks good.

**Comment by songsungkyun** [comment]
@Shredder121 
I added new line. thanks.

**Comment by timowest** [comment]
Sorry to bring this up so late, but I'd convert the BooleanExpression usage back to expression. It's more consistent with similar javadocs (e.g. SimpleExpression) and in most of the cases shouldn't reference implementation types directly.

**Comment by songsungkyun** [comment]
@timowest 
Thanks. I converted them.

**Comment by timowest** [comment]
Looks good

**Comment by johnktims** [comment]
Looks good to me. @Shredder121, it's all yours.

**Comment by Shredder121** [comment]
Yes, looks fine, thanks @songsungkyun!

**Pull request merged**